clients/nym-client/src/clients/mod.rs

```diff
@@ -150,7 +156,7 @@ impl NymClient {
         topology: Topology,
     ) {
         loop {
-            println!("[LOOP COVER TRAFFIC STREAM] - next cover message!");
+            info!("[LOOP COVER TRAFFIC STREAM] - next cover message!");
```

Comment:
I would actually change this (and similar messages) to lower level, debug or possibly even trace